### ISSUE TITLE
loss = nan

### ISSUE THREAD

**Issue by ghost** [chenjun2hao/GIOU_EAST#3]
@chenjun2hao After training the dataset for 101 epochs, suddenly the loss drops to nan, and it can't detect anything.

**Comment by chenjun2hao** [comment]
i meet the same problem with my own dataset, it may be the image_rotate operater problem.
